.github/workflows/create-release-branch.yml

```
@@ -118,8 +118,18 @@ jobs:
           rush build -t @azure/communication-react
         id: buildapifiles
 
+      - name: Check for API changes
+        id: API-check
+        run: |
+          if [[ -z $(git status packages/communication-react -s) ]]
+          then
+            echo "hasChanged=false" >> $GITHUB_OUTPUT
+          else
+            echo "hasChanged=true" >> $GITHUB_OUTPUT
+          fi
+
       - name: Commit beta release snapshots
```

Can you update this name also, confused me a lot reading this
```suggestion
      - name: Commit API File changes
```